airbyte-webapp/public/exclamationPoint.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think we need to create a SVG for this. We already use font awesome, which has [`faExclamationCircle`](https://fontawesome.com/icons/circle-exclamation?s=solid) which pretty much is this SVG, so I think we should just use the font awesome one.